src/platform/packages/shared/kbn-ftr-common-functional-ui-services/services/test_subjects.ts

```diff
@@ -463,4 +463,10 @@ export class TestSubjects extends FtrService {
       }
     );
   }
+
+  public async hasFocus(selector: string) {
```

Comment:
@elastic/appex-qa This is a helper function moved into `testSubjects` from https://github.com/elastic/kibana/pull/230878/files#diff-5e95c55c81ba2791531f64eb4d22a11ca4d2d75b32f406323f1aabc3714b829dL25-L29 so it's reusable in other tests

Reply:
thank you!